content/rma_intro.tex

```diff
@@ -16,31 +16,65 @@
 one-sided communication interfaces via \Celev{} generic selection 
 (\Celev{} \S6.5.1.1\footnote{Formally, the \Celev{} specification is ISO/IEC 9899:2011(E).})
 for block, scalar, and block-strided put and get communication. 
-Such type-generic routines are supported for the ``standard \ac{RMA} types’’
-identified in Table \ref{stdrmatypes}.
+Such type-generic routines are supported for the
+\newtext{\Celev{}
+``standard integer types'' (\Celev{} \S6.2.5.7),
+``real floating types'' (\Celev{} \S6.2.5.10),
+``exact-width integer types'' (\Celev{} \S7.20.1.1),
+``integer types capable of holding object pointers'' (\Celev{} \S7.20.1.4),
+and the integer types \textit{size\_t} and \textit{ptrdiff\_t} defined in
+stddef.h (\Celev{} \S7.19), which are collectively called the}
+``standard \ac{RMA} types''
+\newtext{and are listed} \oldtext{identified} in Table \ref{stdrmatypes}.
+\newtext{Implementations may optionally support additional types.}
```

Comment:
Same comment as above; I don't think line 29 adds anything of substance and it is potentially in conflict with restrictions from other parts of the spec.